main/src/ca/uwaterloo/flix/language/phase/Typer.scala

```diff
@@ -839,7 +946,7 @@ object Typer extends Phase[ResolvedAst.Root, TypedAst.Root] {
           val caze = decl.cases(tag)
 
           // Instantiate the type scheme of the case.
-          val (_, tagType) = Scheme.instantiate(caze.sc, InstantiateMode.Flexible)
+          val (tagConstrs, tagType) = Scheme.instantiate(caze.sc, InstantiateMode.Flexible)
```

Comment:
Is this always empty or what?

Reply:
Yes, I had added that back when I was working with tconstrs in enums, but it should be always empty.